templates/controllers/grid/navigationMenus/form/navigationMenuItemsForm.tpl

```diff
@@ -48,7 +48,11 @@
 				{fbvFormSection title="manager.navigationMenus.form.path" for="path" required="true"}
 					{fbvElement type="text" id="path" value=$path required="true"}
 					<p>
-						{url|replace:"REPLACEME":"%PATH%"|assign:"exampleUrl" router=$smarty.const.ROUTE_PAGE context=$currentContext->getPath() page="navigationMenu" op="view" path="REPLACEME"}
+                        {if $currentContext}
+                            {url|replace:"REPLACEME":"%PATH%"|assign:"exampleUrl" router=$smarty.const.ROUTE_PAGE context=$currentContext->getPath() page="navigationMenu" op="view" path="REPLACEME"}
```

Comment:
Are you sure this is necessary? When `context` is not specified, the current context's path should be used already.

Reply:
(I think an equivalent but shorter fix would just be to remove the `context=...` parameter from the `{url ...}` call that was originally here.)

Reply:
You are right @asmecher ... I changed it following your suggestion.